internal/infrastructure/kubernetes/infra.go

```diff
@@ -17,8 +17,16 @@ import (
 
 	egv1a1 "github.com/envoyproxy/gateway/api/v1alpha1"
 	"github.com/envoyproxy/gateway/internal/envoygateway/config"
+	"github.com/envoyproxy/gateway/internal/infrastructure/kubernetes/proxy"
+	"github.com/envoyproxy/gateway/internal/infrastructure/kubernetes/ratelimit"
 )
 
+//nolint:gocritic
```

Comment:
why this's needed?

Reply:
https://github.com/envoyproxy/gateway/actions/runs/11065043979/job/30743811476

Reply:
https://github.com/envoyproxy/gateway/actions/runs/11065043979/job/30743811476 The action report but maybe it is not needed.